akka-stream/src/main/scala/akka/stream/impl/ActorMaterializerImpl.scala

```diff
@@ -215,6 +212,9 @@ private[akka] object StreamSupervisor {
   def props(settings: ActorMaterializerSettings, haveShutDown: AtomicBoolean): Props =
     Props(new StreamSupervisor(settings, haveShutDown)).withDeploy(Deploy.local)
 
+  private val actorName = SeqActorName("StreamSupervisor")
+  def nextName(): String = actorName.next()
+
   final case class Materialize(props: Props, name: String)
     extends DeadLetterSuppression with NoSerializationVerificationNeeded
 
```

Comment:
I think FlowOps.deprecatedAndThenMat can be now removed (but not yet deprecatedAndThen since stream-o-streams needs those)